Fortran/gfortran/regression/DisabledFiles.cmake

```diff
@@ -119,6 +119,23 @@ file(GLOB UNSUPPORTED_FILES CONFIGURE_DEPENDS
   unlimited_polymorphic_14.f90
   # Unsupported predefined macro: __TIMESTAMP__
   wdate-time.F90
+
+  # This test occasionally fails. It checks that two arrays initialized with
+  # random real numbers, converted to integers, are not identical. This might be
+  # the case in gfortran, but in flang, because of the precision loss and random
+  # nature of the test, it is possible, for two "randomly initialized" arrays to
+  # be identical. Unless something in flang's implementation of the RANDOM_*
+  # intrinsics changes, this test will be unsupported.
```

Comment:
nit: this should be fixed soon in flang and so the wording should change here to make it clear there isn't a bug in flang compared to gfortran, the test is just inherently unreliable

See https://github.com/llvm/llvm-project/pull/106250

Reply:
LGTM, thanks

Reply:
LGTM.